provision-contest/ansible/Makefile

```
@@ -35,6 +35,15 @@ CODEONLY_ROLES=$(addprefix codeonly-,domserver judgehost admin)
 $(CODEONLY_ROLES): codeonly-%:
 	ansible-playbook -i hosts --tags pretask,domjudge_build $*.yml
 
+powerloss:
+	ansible-playbook -i hosts emergency.yml --tags powerloss
+
+lockdown:
+	ansible-playbook -i hosts emergeny.yml --tags full_lockdown
```

```suggestion
	ansible-playbook -i hosts emergency.yml --tags full_lockdown
```

Same typo for the `lockdown-force` makefile target below this one too.